build/source/driver/multi_driver.f90

```diff
@@ -432,9 +453,12 @@ program multi_driver
 ! (5d) compute derived model variables that are pretty much constant for the basin as a whole
 ! *****************************************************************************
 
+!reset startHRU for a full simulation
+if (strtHRU < 1) strtHRU = 1
```

Comment:
See comment above -- if full simulation (no parallelization) then `startHRU=1` and there is no need for this here. The unknown is `nHRU=integerMissing`, which is populated in `allocspace.f90`